server/profileFormSubmission.js

```diff
@@ -8,25 +11,52 @@ const pool = new Pool({
     port: process.env.port,
 });
 
+s3.config.update({
+    accessKeyId: "AKIA47HST523YVS7DQNW",  
+    secretAccessKey: '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',
```

Comment:
I assume we'll be storing the accesskeys in a more secure way :P but just calling it out!

Reply:
Yeah, that was unbelievably dumb of me. Just revoked access to those keys in AWS and regen'd them. Still can't believe I did that.